### DISCUSSION TITLE
Is it possible to "emulate" a yubikey in emergency case

### DISCUSSION THREAD

**Discussion by networkException** [keepassxreboot/keepassxc#6138] [Q&A] [answered]
Hello,

I am wondering if it would be possible to access a KeePass database in case my yubikey gets lost or breaks if I know the
code (stored on the yubikey) that is being used for the HMAC response using some kind of software emulation or if the only way to regain access is to purchase a new yubikey and write the code to it (as mentioned [here](https://keepassxc.org/docs/#faq-yubikey-broken-yubikey)).

Thank you for reading and this wonderful application you are providing to the community!

**Comment by droidmonkey** [comment, marked as answer]
Sure is! https://www.github.com/keepassxreboot/keepassxc/tree/develop/utils%2Fkeepassxc-cr-recovery%2FREADME.md

**Reply by networkException** [reply]
Amazing thank you!
